src/components/EmptyView.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { StyleProp, StyleSheet, Text, View, ViewStyle } from 'react-native'
+import fontStyles from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+
+interface Props {
+  style?: StyleProp<ViewStyle>
+  text: string
+  children?: React.ReactNode
+}
+
+/**
+ * A component to render an empty state with a message and optional children.
+ */
+export default function EmptyView(props: Props) {
```

Comment:
i don't have a big problem with this component but do you think it's worth abstracting out into the general components folder? my gut reaction to this component is that we'll forget that it exists or want to use it but the padding it comes with is doesn't match the design...

Reply:
Yes my goal is to try to force us to think about empty / error states and display them consistently.
I hope we can get to that :D